modules/serverpod_auth/serverpod_auth_server/pubspec.yaml

```diff
@@ -7,22 +7,20 @@ version: 2.4.0
 repository: https://github.com/serverpod/serverpod
 
 environment:
-  sdk: '>=3.3.0 <4.0.0'
+  sdk: ">=3.3.0 <4.0.0"
 
 dependencies:
   serverpod: 2.4.0
   serverpod_serialization: 2.4.0
   serverpod_shared: 2.4.0
   crypto: ^3.0.2
-  email_validator: '>=2.1.17 <4.0.0'
   openid_client: ^0.4.8
   meta: ^1.8.0
-  googleapis: '>=11.0.0 <14.0.0'
+  googleapis: ">=11.0.0 <14.0.0"
   googleapis_auth: ^1.6.0
-  http: '>=1.0.0 <2.0.0'
+  http: ">=1.0.0 <2.0.0"
   image: ^4.0.15
   jose: ^0.3.3
-  pointycastle: ^3.7.4
```

Comment:
Check client generation when we add the email auth & endpoint dependencies here.

Would the client be properly generated? Likely at least the `protocol.dart` would exhibit the same issue because the model class `EmailLogin` from `email_auth` is used without being a direct import.